### DISCUSSION TITLE
Activation of your first VM in GCE, with a Google coupon

### DISCUSSION THREAD

**Discussion by drosofff** [ARTbio/AnalyseGenome#24] [General]
Please post below the result of your VM activation, as described in 
https://artbio.github.io/startbio/AnalyseGenomes_2023/Google_cloud_Account/

**Comment by drosofff** [comment]
drosofff@bare-galaxy:~$ lsb_release -a && lscpu | grep 'CPU(s):' && free -h | grep 'Mem:' && df -h | grep '/$'
    No LSB modules are available.
    Distributor ID: Ubuntu
    Description:    Ubuntu 20.04.6 LTS
    Release:        20.04
    Codename:       focal
    CPU(s):                             8
    NUMA node0 CPU(s):                  0-7
    Mem:           31Gi       284Mi        30Gi       0.0Ki       509Mi        30Gi
    /dev/root       194G  1.9G  192G   1% /

**Comment by slecrom** [comment]
```
root@test-1:~# lsb_release -a && lscpu | grep 'CPU(s):' && free -h | grep 'Mem:' && df -h | grep '/$'

No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.6 LTS
Release:        20.04
Codename:       focal
CPU(s):                             4
NUMA node0 CPU(s):                  0-3
Mem:           15Gi       232Mi        14Gi       0.0Ki       496Mi        15Gi
/dev/root        49G  1.9G   47G   4% /
```

**Comment by AstridCanal** [comment]
```
astrid_canalbizeul@bare-galaxy:~$ lsb_release -a && lscpu | grep 'CPU(s):' && free -h | grep 'Mem:' && df -h | grep '/$'
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.6 LTS
Release:        20.04
Codename:       focal
CPU(s):                             8
NUMA node0 CPU(s):                  0-7
Mem:           31Gi       282Mi        30Gi       0.0Ki       508Mi        30Gi
/dev/root       194G  1.9G  192G   1% /
```

**Reply by drosofff** [reply]
Bravo ! J'ai juste reformaté votre post avec ``` avant et après le block de code 😄 
Première ! :tada:

**Reply by AstridCanal** [reply]
Merci! J'ai tenté avec un seul ` mais ça faisait tout aussi moche que la toute première version

**Comment by yeloett** [comment]


**Reply by drosofff** [reply]
ça, c'est moche !

**Reply by slecrom** [reply]
C'est de ma faute, j'ai demandé une copie d'écran et pas de copier/coller le résultat.

**Reply by drosofff** [reply]
Les copies d'écran, en informatique, c'est moche (et ce n'est pas bon pour la planète) 😄

**Comment by yeloett** [comment]
```
malumsdrug@bare-galaxy:~$ lsb_release -a && lscpu | grep 'CPU(s):' && free -h | grep 'Mem:' && df -h | grep '/$'
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.6 LTS
Release:        20.04
Codename:       focal
CPU(s):                             8
NUMA node0 CPU(s):                  0-7
Mem:           31Gi       281Mi        30Gi       0.0Ki       508Mi        30Gi
/dev/root       194G  1.9G  192G   1% /
```

**Reply by yeloett** [reply]
Je l'ai refait du coup en faisant un copier-coller cette fois !

**Reply by drosofff** [reply]
Merci ! trop beau !

**Comment by zoeguilbert** [comment]
zoe_guilbert_20@bare-galaxy:~$ lsb_release -a && lscpu | grep 'CPU(s):' && free -h | grep 'Mem:' && df -h | grep '/$'
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.6 LTS
Release:        20.04
Codename:       focal
CPU(s):                             8
NUMA node0 CPU(s):                  0-7
Mem:           31Gi       276Mi        30Gi       0.0Ki       509Mi        30Gi
/dev/root       194G  1.9G  192G   1% /
zoe_guilbert_20@bare-galaxy:~$

**Reply by zoeguilbert** [reply]
lsb_release -a && lscpu | grep 'CPU(s):' && free -h | grep 'Mem:' && df -h | grep '/$'
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.6 LTS
Release:        20.04
Codename:       focal
CPU(s):                             8
NUMA node0 CPU(s):                  0-7
Mem:           31Gi       276Mi        30Gi       0.0Ki       509Mi        30Gi
/dev/root       194G  1.9G  192G   1% /

**Reply by drosofff** [reply]
👍 8 cpus, Ubuntu 20.04 :tada:

**Comment by Salimaiche** [comment]
![image](https://github.com/ARTbio/AnalyseGenome/assets/150276101/821d9a61-9297-4d6f-8a10-7e8f18f69beb)

**Reply by drosofff** [reply]
Vous avez pris une photo d'écran avec votre smartphone ? :cry:
On voit quand même qu'il n'y a que 2 CPUs. Vous n'avez pas suivi la consigne...
L'OS du système n'est pas correct non plus...

**Reply by Salimaiche** [reply]
Bonjour, non c'est un screen mac qui a subit une compression lors du copier coller sur cette plateforme. J'ai rencontré des difficultés, le coupon de 50$ ne s'affiche pas malgré que j'ai utilisé le code envoyé. On me propose aussi 300$ offert sans facturation à condition je met mes coordonnées bancaires (une sorte de période d'essaie). Dans facturation y a 0$ mais j'arrive quand même à lancer la VM. Je me suis connecté avec mon compte Google, je n'ai pas eu à crée de nouveau compte. Je ne comprend pas pourquoi j'ai que 10 giga de dispo, c'est effectivement insuffisant pour le TP je pense. 

Cordialement 
Aiche Salim

**Reply by slecrom** [reply]
Bonjour @Salimaiche 
Je vous propose que l'on regarde cela ce matin à la suite de la conférence. Est-ce qu'un message d'erreur s'affiche quand vous utilisez le coupon ?

**Comment by JulieBirgel** [comment]
![image](https://github.com/ARTbio/AnalyseGenome/assets/150080641/d2bb871d-f26c-4a9c-9043-4f62a7bb5b1f)

**Reply by JulieBirgel** [reply]
Oups, je viens de voir votre message @drosofff, désolée pour la copie écran ! Je viens de supprimer la VM... :/

**Reply by drosofff** [reply]
Mouais... C'est surtout 8 cpus qui m'auraient fait plaisir, et 200G de disque, pendant qu'on y est....

> Configuration de la machine
> USAGE général
> Série: E2
> Type de machine: PRÉDEFINI ▶ Standard ▶ e2-standard-8

`/dev/root            49G` ▶ `/dev/root            200G` please. Pour le test, ce n'est pas grave, mais pour le TP, 50G seront trop justes

**Comment by laureasselin** [comment]
```
lsb_release -a && lscpu | grep ‘CPU(s):’ && free -h | grep ‘Mem:’ && df -h | grep ‘/$’
No LSB modules are available.
Distributor ID: Debian
Description:    Debian GNU/Linux 11 (bullseye)
Release:        11
Codename:       bullseye
CPU(s):                             2
NUMA node0 CPU(s):                  0,1
Mem:           3.8Gi       104Mi       3.6Gi       0.0Ki       150Mi       3.5Gi
/dev/sda1       9.7G  2.0G  7.2G  22% /
```

**Reply by drosofff** [reply]
Merci.....

Mais c'est une machine sour `Debian GNU/Linux 11`
avec `2 CPUs`
et seulement `10G` de disque.

Donc pour la prise en main d'une VM, OK, Mais pour la suite, si vous ne respectez pas les consignes, ça ne va pas le faire...

**Comment by CoralineBernachot** [comment]
```grep '/$'
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.6 LTS
Release:        20.04
Codename:       focal
CPU(s):                             8
NUMA node0 CPU(s):                  0-7
Mem:           31Gi       287Mi        30Gi       0.0Ki       508Mi        30Gi
/dev/root       194G  1.9G  192G   1% /
```

**Comment by lin-ch1** [comment]
lsb_release -a && lscpu | grep 'CPU(s):' && free -h | grep 'Mem:' && df -h | grep '/$'
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.6 LTS
Release:        20.04
Codename:       focal
CPU(s):                             8
NUMA node0 CPU(s):                  0-7
Mem:           31Gi       280Mi        30Gi       0.0Ki       507Mi        30Gi
/dev/root       194G  1.9G  192G   1% /

**Comment by AlixMartin1** [comment]
alix_martin2001@instance-alix-martin:~$ lsb_release -a && lscpu | grep 'CPU(s):' && free -h | grep 'Mem:' && df -h | grep '/$'
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.6 LTS
Release:        20.04
Codename:       focal
CPU(s):                             2
NUMA node0 CPU(s):                  0,1
Mem:          3.8Gi       174Mi       3.2Gi       0.0Ki       505Mi       3.4Gi
/dev/root       194G  1.9G  192G   1% /

**Reply by drosofff** [reply]
Merci, et attention quand même:

> CPU(s): 2
> Mem: 3.8Gi 174Mi 3.2Gi 0.0Ki 505Mi 3.4Gi

Cela ne sera pas suffisant pour le TP à venir !

**Comment by LaurineBourel** [comment]
laurine_bourel77@bare-galaxy:~$ lsb_release -a && lscpu | grep 'CPU(s):' && free -h | grep 'Mem:' && df -h | grep '/$'
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.6 LTS
Release:        20.04
Codename:       focal
CPU(s):                             8
NUMA node0 CPU(s):                  0-7
Mem:           31Gi       286Mi        29Gi       0.0Ki       1.2Gi        30Gi
/dev/root       194G  2.3G  192G   2% /

**Comment by MarieMassier** [comment]
massier_marie@bare-galaxy:~$ lsb_release -a && lscpu | grep 'CPU(s):' && free -h | grep 'Mem:' && df -h | grep '/$'
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.6 LTS
Release:        20.04
Codename:       focal
CPU(s):                             8
NUMA node0 CPU(s):                  0-7
Mem:           31Gi       276Mi        30Gi       0.0Ki       508Mi        30Gi
/dev/root       194G  1.9G  192G   1% /

**Comment by claudiamart1s** [comment]
claudiamartins_adp@bare-galaxy:~$ lsb_release -a && lscpu | grep 'CPU(s):' && free -h | grep 'Mem:' && df -h | grep '/$'
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.6 LTS
Release:        20.04
Codename:       focal
CPU(s):                             8
NUMA node0 CPU(s):                  0-7
Mem:           31Gi       288Mi        30Gi       0.0Ki       509Mi        30Gi
/dev/root       194G  1.9G  192G   1% /

**Comment by JulieBirgel** [comment]
juliebirgel1@bare-galaxy:~$ lsb_release -a && lscpu | grep 'CPU(s):' && free -h | grep 'Mem:' && df -h | grep '/$'
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.6 LTS
Release:        20.04
Codename:       focal
CPU(s):                             8
NUMA node0 CPU(s):                  0-7
Mem:           31Gi       287Mi        30Gi       0.0Ki       507Mi        30Gi
/dev/root       194G  1.9G  192G   1% /

**Comment by SabineVautier** [comment]
sabinevtr@bare-galaxy:~$ lsb_release -a && lscpu | grep 'CPU(s):' && free -h | grep 'Mem:' && df -h | grep '/$'
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.6 LTS
Release:        20.04
Codename:       focal
CPU(s):                             8
NUMA node0 CPU(s):                  0-7
Mem:           31Gi       288Mi        30Gi       0.0Ki       508Mi        30Gi
/dev/root       194G  1.9G  192G   1% /

**Comment by camillejarry** [comment]
camillejarry12@instance-1:~$ lsb_release -a && lscpu | grep 'CPU(s):' && free -h | grep 'Mem:' && df -h | grep '/$'
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.6 LTS
Release:        20.04
Codename:       focal
CPU(s):                             8
NUMA node0 CPU(s):                  0-7
Mem:           31Gi       293Mi        30Gi       0.0Ki       508Mi        30Gi
/dev/root       194G  1.9G  192G   1% /
